### DISCUSSION TITLE
Just Vogue Tv - Worldwide Fashion and Premium LifeStyle Channel

### DISCUSSION THREAD

**Discussion by efeyontan** [iptv-org/community#975] [Show and tell]
Here is the player link :
https://tivi191.ozelip.com/broadcaster/just/stream/player

Ofiicial Website of The Channel : 
https://www.justvoguetv.com

**Comment by BellezaEmporium** [comment]
Well, the link is here : https://tivi191.ozelip.com:3873/stream/play.m3u8

**Reply by efeyontan** [reply]
how we can add this tv chanell to the database?

**Reply by BellezaEmporium** [reply]
You can make an issue at [this address](https://github.com/iptv-org/database/issues/new?assignees=&labels=channels%3Aadd&projects=&template=__channels_add.yml&title=Add%3A+), mentioning a channel addition. Be as complete as possible. If you have doubts filling certain things, only fill the necessary blocks.

If that may help, it seems it's a channel originating from Dubai, as its owners are MRT INTERNATIONAL MEDIA L.L.C-FZ.

Though I have *high* doubts over their terms of use & policy. An "-18 years of age" channel only depicting fashion shows (apparently depicts violence, sex and substance abuse) ?
I don't think it's a "Fashion TV Midnight Secrets" level, but knowing that citizens of that country are highly regarding on that kind of content, seems to be a copy and paste from a sort of camgirl site or something (VIP memberships & members over a single stream of fashion shows, lol...)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19895
